arm/Microsoft.ServiceBus/namespaces/.parameters/parameters.json

```diff
@@ -159,6 +159,9 @@
                     "service": "namespace"
                 }
             ]
+        },
+        "allowTrustedServices": {
+            "value": false
```

Comment:
I know the default is `true` - but I'd suggest to set it to true in the param file too, to set an example and stuff ;)

Reply:
Originally set to false to ensure functionality was achieved in pipeline deployment - Will change

Reply:
updated in commit #ec1d408